src/commands/server/news/remove.ts

```diff
@@ -36,24 +32,25 @@ export default class Remove {
     const providedId = interaction.options.getString("id");
     const member = guild.members.cache.get(user.id);
 
-    if (!member.permissions.has(PermissionsBitField.Flags.ManageGuild)) {
-      return await interaction.followUp({
-        embeds: [errorEmbed("You need **Manage Server** permissions to delete news.")],
-      });
-    }
+    if (!member.permissions.has(PermissionsBitField.Flags.ManageGuild)) return await interaction.followUp({
+      embeds: [errorEmbed("You need **Manage Server** permissions to delete news.")],
+    });
 
-    const embed = new EmbedBuilder().setTitle("✅ • News deleted!").setColor(genColor(100));
+    const embed = new EmbedBuilder()
+      .setTitle("✅ • News deleted!")
+      .setColor(genColor(100));
+
+    const subscribedChannel = await newsTable
+      ?.get(`${guild.id}.channel`)
```

Comment:
```suggestion
    const subscribedChannel = await newsTable?
      .get(`${guild.id}.channel`)
```
That character belongs there I think ;)